packages/core/backend/sirius-components-collaborative/pom.xml

```diff
@@ -40,6 +40,10 @@
 	</distributionManagement>
 
 	<dependencies>
+		<dependency>
+			<groupId>com.graphql-java</groupId>
+			<artifactId>graphql-java</artifactId>
+		</dependency>
```

Comment:
Why? The collaborative part is supposed to be independent from that unless I'm mistaken

Reply:
Removed